### DISCUSSION TITLE
Public email | Symbiota users

### DISCUSSION THREAD

**Discussion by Kantoborgy** [Symbiota/community-board#168] [Problems/Bugs]
Hi all,
There is some way to prevent anyone from seeing the email of symbiota users, for example when executing the link https://...symbiotaportalx/portal/profile/userprofile.php?userid=2.    I have a spam problem on my e-accounts.

Thanks in advance

**Comment by Kantoborgy** [comment]
As a Symbiota user's email is public, it also allows anyone to request a change of the user's password. That is, they cannot access the user account but they can change the key. Using email to send spam  :-(

**Comment by themerekat** [comment]
Thanks for bringing up these concerns, @kantobogy ! We'll investigate.

**Reply by mickley** [reply]
@themerekat This is a duplicate of #115. If this vulnerability is being exploited, it is well worth closing.

**Reply by themerekat** [reply]
Currently working on it!
